packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -327,6 +328,50 @@ export class MirrorNodeClient {
             [400, 404],
             requestId);
     }
+    /*******************************************************************************
+     * To be used to make paginated calls for the account information when the 
+     * transaction count exceeds the constant MIRROR_NODE_QUERY_LIMIT.
+     *******************************************************************************/
+    public async getAccountPaginated(url: string, requestId?: string) {
+        const queryParamObject = {};
+        const accountId = this.extractAccountIdFromUrl(url, requestId);
+        const params = new URLSearchParams(url.split('?')[1]);
+        
+        this.setQueryParam(queryParamObject, 'limit', constants.MIRROR_NODE_QUERY_LIMIT);
+        this.setQueryParam(queryParamObject, 'timestamp', params.get('timestamp'));
+        const queryParams = this.getQueryParams(queryParamObject);
+
+        return this.getPaginatedResults(
+            `${MirrorNodeClient.GET_ACCOUNTS_ENDPOINT}${accountId}${queryParams}`,
+            MirrorNodeClient.GET_ACCOUNTS_ENDPOINT,
+            'transactions',
+            [400, 404],
```

Comment:
I missed this on [PR](https://github.com/hashgraph/hedera-json-rpc-relay/pull/1163) 
But we no longer need to send the acceptedResponseStatusCodes to the client.

It would be nice if you can remove that param from here and from the `getPaginatedResults` so it does not raise confusion.

on the details of `getPaginatedResults` you will notice that is not doing anything that passing that parameter to itself without actually using it.

Reply:
Updated.